src/redux/categories/categoriesSlices.js

```diff
@@ -14,4 +14,4 @@ const categoriesSlices = createSlice({
 });
 
 export const { checkStatus } = categoriesSlices.actions;
-export default categoriesSlices.reducer;
+export default categoriesSlices;
```

Comment:
- [ ] You did a great job setting up the slices. :+1: I believe we should make a default export of the reducer from the slice. Currently, you are exporting the entire slice instead of the reducer. Please update it as follows 😎️

```
export default categoriesSlices.reducer;
```

- [ ] Please also ensure the `checkStatus` function always returns the text `Under construction`. You can take the `state` as the function argument and return `state.status` to achieve it. 🤝️